docs/reference/overview/excel-add-ins-reference-overview.md

```diff
@@ -8,56 +8,38 @@ localization_priority: Priority
 
 # Excel JavaScript API overview
 
-> [!NOTE]
-> TO DO: Kim will be updating the structure and content of this article before the new TOC structure and landing pages are published to the live doc set.
+An Excel add-in interacts with objects in Excel by using the JavaScript API for Office, which includes two JavaScript object models:
 
-You can use the Excel JavaScript API to build add-ins for Excel 2016 or later. The following list shows the high-level Excel objects that are available in the API. Each object page link contains a description of the properties, events, and methods that are available on the object. Explore the links from the menu to learn more.
+* **Excel JavaScript API**: Introduced with Office 2016, the [Excel JavaScript API](/javascript/api/excel) provides strongly-typed objects that you can use to access worksheets, ranges, tables, charts, and more. 
 
-Some of the core Excel objects are listed below for convenience:
+* **Common APIs**: Introduced with Office 2013, the [Common API](/javascript/api/office) can be used to access features such as UI, dialogs, and client settings that are common across multiple types of host applications such as Word, Excel, and PowerPoint.
 
-- [Workbook](/javascript/api/excel/excel.workbook): The top-level object that contains related workbook objects such as worksheets, tables, ranges, etc. It also can be used to list related references.
+This section of the documentation focuses on the Excel JavaScript API, which you'll use to develop the majority of functionality in add-ins that target Excel 2016 or later. For information about the Common API, see [JavaScript API for Office](../javascript-api-for-office.md). 
 
-- [Worksheet](/javascript/api/excel/excel.worksheet): Represents a worksheet in a workbook.
-  - [WorksheetCollection](/javascript/api/excel/excel.worksheetcollection): A collection of the **Worksheet** objects in a workbook.
-  - [WorksheetProtection](/javascript/api/excel/excel.worksheetprotection): Represents the protection of a **Worksheet** object.
+## Learn programming concepts
 
-- [Range](/javascript/api/excel/excel.range): Represents a cell, a row, a column, or a selection of cells containing one or more contiguous blocks of cells.
-  - [ConditionalFormat](/javascript/api/excel/excel.conditionalformat): An object defining a rule and a format applied to the range when the rule's condition is met.
-  - [DataValidation](/javascript/api/excel/excel.datavalidation): An object that restricts user input to a range based on a variety of criteria.
-  - [RangeSort](/javascript/api/excel/excel.rangesort): Represents a object that manages sorting operations on a range.
+Review the following articles for information about important programming concepts:
```

Comment:
I'm not sure how I feel about the command to "review". I like that it gives users an instruction, but it implies they've read or otherwise been exposed to the information before.